cmd/bootmap.c

```diff
@@ -0,0 +1,46 @@
+#include <common.h>
+#include <command.h>
+#include <console.h>
+
+static int do_get_active_bootmap(cmd_tbl_t * cmdtp, int flag, int argc, char * const argv[])
+{
+	void *addr;
+	struct bootmap_switch {
+		u32 address;
+		u32 size;
+	} *bms;
+
+	if (argc != 2)
+		return CMD_RET_USAGE;
+
+	addr = (void *)simple_strtoul(argv[1], NULL, 16);
```

Comment:
Since the `addr` is de-referenced later, there is a check for `NULL` pointer missing.

Something like this would be good to have:
```c
if(!addr) {
    printf("Bootmap address is NULL!\n");
    return CMD_RET_USAGE;
}
````